### ISSUE TITLE
Profiles are not completed after --profile=

### ISSUE THREAD

**Issue by Coacher** [2ion/mpv-bash-completion#12]
Hello.

It would be nice to have profile completion.
List of profiles is available via `mpv --profiles=help` in recent mpv versions.

**Comment by 2ion** [comment]
Profile completion already exists, with `--profile <TAB>`, i.e. without = character (see #7).

What does not work is completion of profiles defined in configuration files as gen.lua calls mpv with --no-config as per your request in #3, so that for example when in user-mode, --profiles has the list
```
Available profiles:
        protocol.ytdl
        protocol.https
        protocol.http
        opengl-hq
        encoding
        libmpv
        builtin-pseudo-gui
        pseudo-gui
        default
```
with --no-config, this becomes
```
Available profiles:
        opengl-hq
        encoding
        libmpv
        builtin-pseudo-gui
        pseudo-gui
        default
```
and these will be the profiles offered for completion (in this case, missing profile.\*).

Is this good enough? Otherwise I'd need to call `mpv --profile=help` from the generated completion function itself in order to detect user-defined profiles.

**Comment by Coacher** [comment]
> What does not work is completion of profiles defined in configuration files as gen.lua calls mpv with --no-config as per your request in #3

I see. I still think mpv should be called with `--no-config` during completion generation.
Having profiles hardcoded into completion file defies their purpose anyway, since they are dynamic.

> Otherwise I'd need to call mpv --profile=help from the generated completion function itself in order to detect user-defined profiles.

I believe this is the right way to implement profiles completion. This is a very minor feature though, I can live without it.

**Comment by 2ion** [comment]
The feature is now available in 3.3.12.

**Comment by Coacher** [comment]
Works fine here. Thank you very much!
